src/App.js

```diff
@@ -28,7 +28,7 @@ class App extends React.Component {
 
   notifyPokemonIsLoaded(pokemon) {
     this.setState({
-      pokemons: [...this.state.pokemons, pokemon],
+      pokemons: this.sortPokemons([...this.state.pokemons, pokemon]),
```

Comment:
I just moved the sorting to this class, until we find a better way to do so, to this component to ensure the whole logic related to the list of pokemons is related to this source of truth.